README.md

```diff
@@ -0,0 +1,82 @@
+# AxonServer API Specification
+
+This module contains the protobuf definitions of the services and messages provided by
+AxonServer.
+
+The API definitions are structured around 5 files:
+- `command.proto` - contains the service and message definitions specific to the dispatching and handling of commands.
+- `common.proto` - contains messages commonly used throughout the services
```

Comment:
suggestion: order from general to specific so `common.proto`, `control.proto`, `command.proto`, ...